toxiproxy/introduce_latency/introduce_latency_experiment.json

```diff
@@ -0,0 +1,86 @@
+{
+    "version": "1.0.0",
+    "title": "The application should stay within tolerance when dependency responses slow down.",
+    "description": "Uses Toxiproxy to manipulate the connections to a service",
+    "tags": [
+        "platform:toxiproxy",
+        "service:any"
+    ],
+    "configuration": {
+        "service_url": {
+            "type": "env",
+            "key": "APPLICATION_ENTRYPOINT_URL"
+        },
+        "toxiproxy_host": {
+            "type": "env",
+            "key": "TOXIPROXY_HOST"
+        },
+        "toxiproxy_proxy_name": {
+            "type": "env",
+            "key": "TOXIPROXY_PROXY_NAME"
+        }
+    },
+    "contributions": {
+        "availability": "high",
+        "reliability": "high",
+        "performability": "high",
+        "safety": "none",
+        "security": "none"
+    },
+    "steady-state-hypothesis": {
+        "title": "System is healthy",
+        "probes": [
+            {
+                "name": "consumer-service-must-still-respond",
+                "type": "probe",
+                "provider": {
+                    "type": "http",
+                    "url": "${service_url}"
+                },
+                "tolerance": 200,
+                "timeout": 1
+            }
+        ]
+    },
+    "method": [
+        {
+            "name": "create_latency_toxic",
+            "provider": {
+                "arguments": {
+                    "for_proxy": "${toxiproxy_proxy_name}",
+                    "toxic_name": "${toxiproxy_proxy_name}-latency",
+                    "latency": 1200,
+                    "jitter": 100
+                },
+                "func": "create_latency_toxic",
+                "module": "chaostoxi.toxic.actions",
+                "type": "python"
+            },
+            "type": "action",
+            "pauses": {
+                "after": 1
+            }
+        },
+        {
```

Comment:
I think the delete_toxic step should only appear in the rollbacks section, so the toxic is in place when the steady-state hypothesis get to run at the end of the experiment